### PR TITLE
Add markup for Header section

### PR DESCRIPTION
**Pull request by franiakat**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
```
Comment (szynszyliszys) [inline index.html]: You are missing a lang attribute. Add `lang="en"` to your `html` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Hello Creative Agency Website.">
+    <title>We're Hello! - creative agency from Warsaw</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li>the offer</li>
```
Comment (szynszyliszys) [inline index.html]: Hm these look as if tehy could be linking to the sections of the page but they don't really match what is there. Anyway, empty `<a href="/">` tags could be added as obviously these are links and styling will be more challenging then ;)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          <li>portfolio</li>
+          <li>about us</li>
+          <li>contact</li>
+        </ul>
+      </nav>
+      <section>
+        <h1>
+          hello!
+          <span>We are creative agency based in Warsaw, Poland
+            <br/>Creating projects for clients all over the world
```
Comment (szynszyliszys) [inline index.html]: I am not a fan of using `<br/>` tag. I would say that this whole span could be a `<p>` with two `<span>` elements inside it. Same in the next sections.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      <section>
+        <span>
+          We always select varied projects,
+          <br/>to step out of our comfort zone.
+        </span>
+      </section>
+    </header>
+    <button>scroll down to find out more about our studio</button>
+  </body>
+</html>
```
Comment (szynszyliszys) [inline index.html:43]: Good job 🎉  😄

File: index.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        </span>
+      </section>
+      <section>
+        <span>
+          We always select varied projects,
+          <br/>to step out of our comfort zone.
+        </span>
+      </section>
+    </header>
+    <button>scroll down to find out more about our studio</button>
```
Comment (szynszyliszys) [inline index.html]: In my opinion it would work better as a link (it will let you use anchor tag later to scroll down to the section of a page).

**Review by szynszyliszys — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (franiakat) [reply index.html]: Sure, it supposed to be the navigation! 😅

**Review by szynszyliszys — APPROVED** [review]
🎉

**Pull request merged**